src/platform_impl/windows/event_loop.rs

```diff
@@ -749,7 +749,7 @@ fn wait_for_messages_impl(
             num_handles,
             raw_handles.as_ptr() as *const _,
             wait_duration_ms,
-            QS_ALLEVENTS,
+            QS_ALLINPUT,
```

Comment:
Should add a comment here to say why we use `QS_ALLINPUT` and not `QS_ALLEVENTS`, otherwise it has a chance of being reverted in the future.